docs/user/migration/external-name-alias.md

```diff
@@ -2,11 +2,10 @@
 
 ## Context
 
-Istio with 1.21 version has changed the behavior of the Service of type ExternalName.
-With the change, the Service of type ExternalName is treated as an alias of the Service that it points to.
-Istio introduced this change to align the way it handles the Service of type ExternalName with the Kubernetes behavior.
-Since it caused some issues for our users, we introduced a new annotation, **disable-external-name-alias**, to disable this change.
-However, use of this parameter is not recommended and if you are using it, please update your configuration according to the following points.
+To align with Kubernetes behavior, Istio with 1.21 version changed the behavior of the Service of type ExternalName.
+This Service is now treated as an alias of the Service that it points to.
```

Comment:
If you accept the previous suggestion, please adjust accordingly throughout the entire document.